CHANGELOG.rst

```diff
@@ -4,6 +4,53 @@ Nutanix.Ncp Release Notes
+- Documentation changes - [Imprv] Add detailed doc for using uuid in modules [[\#433](https://github.com/nutanix/nutanix.ansible/issues/433)]
+- ntnx_prism_vm_inventory - [Imprv] Add support for retrieving all VMs without specifying length in inventory plugin [[\#651](https://github.com/nutanix/nutanix.ansible/issues/651)]
+- ntnx_prism_vm_inventory - [Imprv] Make changes to include project_reference in dynamic inventory for groupping [[\#500](https://github.com/nutanix/nutanix.ansible/issues/500)]
+- ntnx_vms_v2 - [Imprv] add functionality uefi boot_order to module ntnx_vms_v2 [[\#579](https://github.com/nutanix/nutanix.ansible/issues/579)]
+
+Bugfixes
+--------
+
+- ntnx_acps - [Bug] Fix comparison of old_context_list and update_context_list in module ntnx_acps [[\#475](https://github.com/nutanix/nutanix.ansible/issues/475)]]
+- ntnx_prism_vm_inventory - [Bug] API failure is not in shown while creating dynamic inventory [[\#421](https://github.com/nutanix/nutanix.ansible/issues/421)]
+- ntnx_prism_vm_inventory - [Bug] Results of VMs is not more then 500 by default in module inventory [[\#354](https://github.com/nutanix/nutanix.ansible/issues/354)]
+- ntnx_prism_vm_inventory - [Bug] galaxy.ansible doc for ntnx_prism_vm_inventory is having Documentation Syntax Error. [[\#453](https://github.com/nutanix/nutanix.ansible/issues/453)]
+- ntnx_protection_rules - [Bug] Fix invalid OU check in user_groups module [[\#481](https://github.com/nutanix/nutanix.ansible/issues/481)]
+- ntnx_security_rules - [Bug] Purpose field mandatory to update the security policy from Ansible in module ntnx_security_rules [[\#485](https://github.com/nutanix/nutanix.ansible/issues/485)]
+- ntnx_vmm - [Bug] "not enough positional arguments" in module plugins/modules/ntnx_vmy.py, line 881 [[\#465](https://github.com/nutanix/nutanix.ansible/issues/465)]
+- ntnx_vms - [Bug] Attaching image to existing disk in module ntnx_vms [[\#454](https://github.com/nutanix/nutanix.ansible/issues/454)]
+- ntnx_vms - [Bug] Cannot assign IP address on an unmanaged network in module ntnx_vms [[\#593](https://github.com/nutanix/nutanix.ansible/issues/593)]
+- ntnx_vms_info_v2 - [Bug] Can't fetch all VMs [[\#662](https://github.com/nutanix/nutanix.ansible/issues/662)]
+- ntnx_vms_v2 - [Bug] No disk resizing in module ntnx_vms_v2 [[\#578](https://github.com/nutanix/nutanix.ansible/issues/578)]
+- ntnx_vms_v2 - [Bug] state absent does not respect --check mode in module nutanix_vms [[\#534](https://github.com/nutanix/nutanix.ansible/issues/534)]
+- recovery_plans - [Bug] recovery_plan fails to create in module plugin_modules/prism/recovery_plans.py [[\#515](https://github.com/nutanix/nutanix.ansible/issues/515)]
+- v3 modules - [Bug] "Failed to convert API response into JSON" in all modules of Nutanix.ncp ansible collection [[\#490](https://github.com/nutanix/nutanix.ansible/issues/490)]
+
+New Modules
```

Comment:
New modules should be added at first. This excites users.
1. new modules
2. Bug fixes
3. minor fixes

This should be the good/better approach